numpy/core/src/multiarray/item_selection.c

```diff
@@ -1074,12 +1077,11 @@ PyArray_Choose(PyArrayObject *ip, PyObject *op, PyArrayObject *out,
                 break;
             }
         }
-        memmove(ret_data, PyArray_MultiIter_DATA(multi, mi), elsize);
+        copyswap(ret_data, PyArray_MultiIter_DATA(multi, mi), swap, NULL);
```

Comment:
I'd prefer not adding new `copyswap` usages if we can avoid it since it will cause seg faults with new dtypes. Happy to push a fixup here if you don't have time to look at the other PRs replacing copyswap with a single-element cast.